content/ref/cli/wandb-agent.md

```diff
@@ -15,9 +15,9 @@ Run the W&B agent
 
 | **Option** | **Description** |
 | :--- | :--- |
-| -p, --project | The name of the project where W&B runs created from the   sweep are sent to. If the project is not specified, the run is sent to a project labeled 'Uncategorized'. |
-| -e, --entity | The username or team name where you want to send W&B   runs created by the sweep to. Ensure that the entity you specify already exists. If you don't specify an entity,   the run will be sent to your default entity, which is usually your username. |
-| --count | The max number of runs for this agent. |
+| `-p, --project` | The name of the project where W&B runs created from the   sweep are sent to. If the project is not specified, the run is sent to a project labeled 'Uncategorized'. |
```

Comment:
A nit that might be useful to follow up later if feasible -- these files could be more useful / easier to programmatically parse if the flags were separately marked up.